src/components/jPlayer/keyControl/keyControlContainer.jsx

```diff
@@ -71,7 +71,7 @@ const lifecycle = {
   componentDidMount() {
     document.addEventListener('keydown', this.props.onKeyDown);
   },
-  componentWillUnmount() {
+  componentDidMount() {
```

Comment:
`componentDidMount` is declared twice. This seems like a mistake.

Reply:
Yeah, I find this question after, already committed this change.